### PR TITLE
fix(DCOS-14334): Update CLI setup script

### PR DESCRIPTION
**Pull request by weblancaster**
**DCOS-14334** Well, the title explains a lot.

**before**
screen shot 2017-05-26 at 1 58 33 pm

**after**
screen shot 2017-05-26 at 1 53 23 pm

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: src/js/components/modals/CliInstallModal.js [hunk 1]
```diff
@@ -77,11 +77,10 @@ class CliInstallModal extends React.Component {
...
       `curl ${downloadUrl} -o dcos`,
-      "sudo mv dcos /usr/local/bin",
-      "sudo chmod +x /usr/local/bin/dcos",
-      `dcos config set core.dcos_url ${clusterUrl}`,
-      Hooks.applyFilter("dcosInstallCommandExtraSteps", undefined),
-      "dcos"
+      `sudo mv dcos /usr/local/bin`,
+      `sudo chmod +x /usr/local/bin/dcos`,
+      `dcos cluster setup ${clusterUrl}`,
+      `dco`
```
Comment (MatApple) [inline src/js/components/modals/CliInstallModal.js]: This should be `dcos`.

Also wondering why we have stripped out `Hooks.applyFilter("dcosInstallCommandExtraSteps", undefined),`?

Comment (weblancaster) [inline src/js/components/modals/CliInstallModal.js]: good catch. misspelled addressed.

Now the extra steps part looks like we don't need anymore based on the Jira ticket https://jira.mesosphere.com/browse/DCOS-14334?

Comment (MatApple) [inline src/js/components/modals/CliInstallModal.js]: @weblancaster I'm not sure that is the case. We shouldn't assume a change in implementation and should always question cases like this. If you search for where `dcosInstallCommandExtraSteps` is defined, please note the additional commands that are added and please verify with @tamarrow. Thanks.

**Comment by weblancaster** [comment]
@tamarrow Can you review and confirm we have removed the login command please? 

`dcos auth login`

**Comment by tamarrow** [comment]
LGTM

**Comment by weblancaster** [comment]
No problem @MatApple thanks for the review

**Pull request merged**